etc/kayobe/kolla-image-tags.yml

```diff
@@ -4,7 +4,7 @@
 # where the key is the OS distro and the value is the tag to deploy.
 kolla_image_tags:
   openstack:
-    rocky-9: 2024.1-rocky-9-20240903T113235
+    rocky-9: 2024.1-rocky-9-20241126T102910
```

Comment:
If you've rebuilt all the images, you can delete all the other rocky-9 tags in this file